xaml/inputXML.xaml

```diff
@@ -9,7 +9,11 @@
         WindowStartupLocation="CenterScreen"
         UseLayoutRounding="True"
         WindowStyle="None"
-        Title="Chris Titus Tech's Windows Utility" Height="800" Width="1280">
+        Width="Auto"
+        Height="Auto"
+        MaxWidth="1380"
```

Comment:
```suggestion
        MaxWidth="1280"
```
I think you meant 1280 instead of 1380? either way, using an even smaller resolution would be better, like 1280 by 720 (which should _probably_ (not sure) cover 99% of displays that run windows around the world)

Reply:
@og-mrk, I suggested that as well. DT also uses that as its default window size

Reply:
well, it is "MaxWidth" and "MaxHeight". Normal values are now set to auto to work with every resolution possible. The max values are there to make the utility not too big for high resolution. 1380 is the current biggest size to make it look reasonable.

Reply:
changed it to 1280 anyways